index.js

```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
+      var magenta = 'ffb3ff';
```

Comment:
If we change `FFB3FF` to anything else, this is no longer magenta, lets use a more generic name for the color.

Reply:
But decided on magenta so does this matter?

Reply:
I think it does. If we decide to theme the colors any differently, or add in the options for themed outputs in the future, it's another change we'd have to make. 

I'd suggest like titleColor, headerColor, or something along those lines.

Thoughts?

Reply:
sure. should be done